DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/credit-and-debit-cards/mastercard/mastercard-status-what-does-this-mean-.md

## Declined
If a Mastercard payment of a customer receives the status _declined_, it may be for several reasons. Only the customer can contact their credit card issuer to inquire as to why the credit card payment has been _declined_.

> For questions about credit card payments and/or a payment status (of a credit card transaction), email our Support Team at <support@multisafepay.com>

Reviewer: ```suggestion
For questions about credit card payments or transaction statuses, email the Support Team at <support@multisafepay.com>
```

In most cases the declined reason is shown behind the declined status within the transaction in your MultiSafepay Control. 

| Transaction status | Reason | Description |
------- |----------|---------|

> Declined | transaction not allowed | The fraud filter of MultiSafepay is declining the transaction. Email our Support Team at <support@multisafepay.com> for further information.

Reviewer: ```suggestion
Declined | transaction not allowed | The MultiSafepay fraud filter declined the transaction. For more information, email the Support Team at <support@multisafepay.com> 
```

Declined | Do not honour | This information is not shared with MultiSafepay, only the card holder can retrieve the information at the issuing bank.
Declined | 3D authorisation cancelled | When the [3D Secure](/faq/general/glossary/#3d-secure) verification is incomplete or can not be validated, the payment will be declined.
Declined | Expired card | The credit card is expired.
Declined | Not sufficient funds | The customer has insufficient credit on the credit card to complete the payment.

> Declined | Merchant only accepts 3DS verified cards | "Declined, not verified". Submit your request to our Risk Team at <risk@multisafepay.com> when wanting to accept unverified 3D cards. 

Reviewer: ```suggestion
Declined | Merchant only accepts 3DS verified cards | "Declined, not verified". Email requests to accept unverified 3D cards to the Risk Team at <risk@multisafepay.com> 
```
